tokenLists/fantom.mainnet.tokenlist.json

```diff
@@ -284,16 +280,15 @@
             "_scan": "https://ftmscan.com/token/0x34D33dc8Ac6f1650D94A7E9A972B47044217600b",
             "chainId": 250,
             "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/SMART.png"
+            "logoURI": "https://s2.coinmarketcap.com/static/img/coins/64x64/15540.png"
         },
         {
             "name": "aUSD",
             "symbol": "aUSD",
             "address": "0x41e3dF7f716aB5AF28c1497B354D79342923196a",
             "_scan": "https://ftmscan.com/token/0x41e3dF7f716aB5AF28c1497B354D79342923196a",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://aquarius.fi/assets/200/AUSD200-t.png"
+            "decimals": 18
         },
```

Comment:
why do you remove logo of this token?
I see aquarius link is error but you can update this to our source then user raw github link 
![image](https://user-images.githubusercontent.com/11427213/183458153-cc3a700e-f643-42fc-85a0-cc90fef29a40.png)

Reply:
Updated